tljh/configurer.py

```diff
@@ -27,6 +27,7 @@
         'allowed': [],
         'banned': [],
         'admin': [],
+        'groups': {}
```

Comment:
Since this is only for users, can we call it something like extra_user_groups or something? 

Other than that this looks great :)

Reply:
Sure, *extra_user_groups* makes more sense :)